README.md

```diff
@@ -69,6 +70,10 @@ If you have an existing guide2go setup you may copy the `.yaml` files into the p
 Simply run the cronjob file inside the Docker container  
 `docker exec -it dockername ./cronjob.sh`
 
+## Notes On Channels DVR
+* You have to select MPEG-TS as the stream format
+* The stream only works in a browser - not on Android TV or Android
+
```

Comment:
You could also add a link to the support document for adding an m3u playlist to Channels: https://getchannels.com/docs/channels-dvr-server/how-to/custom-channels/#adding-your-custom-channels